netlify.toml

```diff
@@ -5,7 +5,7 @@
 [build]
   publish = "public"
   base = "website"
-  command = "cd themes/docsy && git submodule update -f --init && cd ../.. && npm install postcss-cli autoprefixer && hugo version && hugo"
+  command = "cd themes/docsy && git submodule update -f --init && cd ../.. && npm install postcss-cli autoprefixer@^9.0.0 && hugo version && hugo"
```

Comment:
For future reference, https://github.com/operator-framework/operator-sdk/pull/3883